Project/src/MakeCall/MakeCall.js

```diff
@@ -81,6 +81,10 @@ export default class MakeCall extends React.Component {
                         }
                     });
                 });
+                this.callAgent.on('incomingCall', e => {
+                    console.log(`incoming call ${e.incomingCall.state}`)
+                    this.setState({call: e.incomingCall});
+                })
```

Comment:
Have you tested incoming calls? I think more changes are required

Reply:
Yes, I've tested incoming call and accept call also works fine.
What additional changes would be?
![image](https://user-images.githubusercontent.com/9585318/107319137-81097a80-6ae1-11eb-974c-74c74e75b6dc.png)
![image](https://user-images.githubusercontent.com/9585318/107319166-941c4a80-6ae1-11eb-81a7-f226b386be21.png)

Reply:
Should be good then. It would be good to also add call.direction === 'Incoming' for showing the accept button in CallCard.js.

Reply:
> Should be good then. It would be good to also add call.direction === 'Incoming' for showing the accept button in CallCard.js.

CallCard already showing 'Accept' button though, where exactly should I put call.direction === 'Incoming' ?